add-column.md

```diff
@@ -18,6 +18,112 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
+
 ## Examples
 
-## See Also
\ No newline at end of file
+### Add a Single Column
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN names STRING;
+~~~ 
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+~~~
+
+### Add Multiple Columns
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN location STRING, ADD COLUMN amount DECIMAL;
+~~~
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
+| location  | STRING            | true  | NULL    | {}        |
+| amount    | DECIMAL           | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+
+~~~
+
+### Add a Non-Null Column with a Default Value
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN interest DECIMAL NOT NULL DEFAULT (DECIMAL '1.3');
+~~~ 
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+~~~
++-----------+-------------------+-------+---------------------------+-----------+
+|   Field   |       Type        | Null  |          Default          |  Indices  |
++-----------+-------------------+-------+---------------------------+-----------+
+| id        | INT               | false | NULL                      | {primary} |
+| balance   | DECIMAL           | true  | NULL                      | {}        |
+| names     | STRING            | true  | NULL                      | {}        |
+| location  | STRING            | true  | NULL                      | {}        |
+| amount    | DECIMAL           | true  | NULL                      | {}        |
+| interest  | DECIMAL           | false | ('1.3':::STRING::DECIMAL) | {}        |
++-----------+-------------------+-------+---------------------------+-----------+
+~~~
+
+### Add a Non-Null Column with Unique Values
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN cust_number DECIMAL UNIQUE NOT NULL;
+~~~ 
+ 
+### Add a Column with Collation
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN more_names STRING COLLATE en;
+~~~ 
+ 
+### Add a Column to a Column Family
+
+~~~ sql
+# Add a Column to a New Column Family
```

Comment:
I agree. Would "Add a column and assign it to new family" be better? Similarly for other headings. Also, I noticed we use stacked headings quite frequently in our documentation. So it's okay if my document has stacked headings as well, right?